Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -140,10 +142,11 @@ public static void HandlePlaySound(Session session, params string[] parameters)
                     if (Enum.IsDefined(typeof(Network.Enum.Sound), sound))
                     {
                         message = $"Playing sound {Enum.GetName(typeof(Network.Enum.Sound), sound)}";
-                        // add the sound to the player queue for everyone to hear
-                        // player action queue items will execute on the landblock
-                        // player.playsound will play a sound on only the client session that called the function
-                        session.Player.ActionApplySoundEffect(sound, session.Player.Guid);
+                        // Lock the player and play the sound...
+                        lock (session.Player)
+                        {
+                            session.Player.PlaySound(sound, session.Player.Guid);
```

Comment:
I don't think so, but keep in mind sound had a object guid attached to it but player is the one that need to hear it, it plays on another object on their client